e2e/fragments/AtTableView.js

```diff
@@ -0,0 +1,20 @@
+/* global inject */
+const { I } = inject();
+const assert = require('assert');
+
+
+module.exports = {
+  _tableRowSelectors: [
+    '.ant-table-wrapper .ant-table-tbody .ant-table-row:nth-child(1) .ant-table-cell:nth-child(1)',
+    '.ant-table-wrapper .ant-table-tbody .ant-table-row:nth-child(2) .ant-table-cell:nth-child(1)',
+    '.ant-table-wrapper .ant-table-tbody .ant-table-row:nth-child(3) .ant-table-cell:nth-child(1)',
+  ],
+
+  async seeSortedTable(value) {
```

Comment:
it's more like just key comparison, so `seeKeys()` would fit better and consistent with other fragments